src/app/static/src/app/store/plotSelections/getters.ts

```diff
@@ -12,5 +15,59 @@ export const getters = {
         } else {
             return filterSelection.selection
         }
+    },
+    chartData: (state: PlotSelectionsState, getters: any, rootState: RootState, rootGetters: any) =>
```

Comment:
This is a bit nasty but felt better than reworking all of the individual getters, this way the hardcoding for the plot is just in this once place which I don't think is too bad. There might be a nicer way to write this though...